db/db_impl/db_impl_open.cc

```diff
@@ -529,6 +536,40 @@ Status DBImpl::Recover(
   return s;
 }
 
+// Initialize the built-in column family for persistent stats. Depending on
+// whether on-disk persistent stats have been enabled before, it may either
+// create a new column family and column family handle or just a column family
+// handle.
+Status DBImpl::InitPersistStatsColumnFamily() {
+  if (persist_stats_cf_handle_) {
+    return Status::OK();
+  }
+  bool persistent_stats_cfd_exists = false;
+  if (versions_->GetColumnFamilySet()->GetColumnFamily(
+          kPersistentStatsColumnFamilyName) != nullptr) {
+    persistent_stats_cfd_exists = true;
+  }
+
+  if (!persistent_stats_cfd_exists) {
+    mutex_.Unlock();
+    ColumnFamilyHandle* handle = nullptr;
+    Status s = CreateColumnFamily(ColumnFamilyOptions(),
```

Comment:
Here still default `ColumnFamilyOptions()` is used without calling `OptimizeForPersistentStats()`.